Northstar.Client/mod/scripts/vscripts/ui/atlas_auth.nut

```diff
@@ -0,0 +1,47 @@
+global function AtlasAuthDialog
+
+void function AtlasAuthDialog()
+{
+	thread AtlasAuthDialog_Threaded()
+}
+
+void function AtlasAuthDialog_Threaded()
+{
+	while ( !NSIsMasterServerAuthenticated() || GetConVarBool( "ns_auth_allow_insecure" ) )
+		WaitFrame()
+
+	if ( GetConVarBool( "ns_auth_allow_insecure" ) )
+		return
+
+	MasterServerAuthResult res = NSGetMasterServerAuthResult()
+
+	// do nothing on successful authentication
+	if ( res.success )
+	    return
+
+	EmitUISound( "blackmarket_purchase_fail" )
+
+	DialogData dialogData
+	dialogData.image = $"ui/menu/common/dialog_error"
+	dialogData.header = Localize( "#AUTHENTICATION_FAILED_HEADER" )
+	dialogData.message = Localize( "#AUTHENTICATION_FAILED_BODY" )
+
+	// if we got a special error message from Atlas, display it
+	if ( res.errorMessage != "" )
+		dialogData.message += "\n" + res.errorMessage
+
+	if ( res.errorCode != "" )
+		dialogData.message += "\n\n" + Localize( "#AUTHENTICATION_FAILED_ERROR_CODE", res.errorCode )
+
+
+	CloseAllDialogs()
+	AddDialogButton( dialogData, "#OK", null )
+	AddDialogButton( dialogData, Localize( "#AUTHENTICATION_FAILED_HELP" ), void function() {
+		// todo: get MS to redirect, so i can use an MS link or something?
+		// also todo: link using # to the error code given, needs work on wiki side of things
+		string link = format( "%s/wiki", GetConVarString( "ns_masterserver_hostname" ) ) 
```

Comment:
```suggestion
		string link = "https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/troubleshooting"
```
we can change this to ``format( "https://r2northstar.gitbook.io/r2northstar-wiki/installing-northstar/troubleshooting#%s, GetErrorLink( error ) `` in the future

Reply:
I couldnt remove it we had this in the discord already just mark as resolved